docs/developer-guide.md

```diff
@@ -38,14 +38,21 @@ And Our travis CI jobs would also run these, so you could also check out the bui
 
 Now cyclone's e2e test would start a local circle server and register local docker host as a worker to circle server. Then it would start a standalone process to run the e2e test cases by sending RESTful requests.
 
-You could run e2e test by typing:
+If you deploy with docker, you could run e2e test by typing:
```

Comment:
Does the e2e test already support to be run by cyclone? If so, maybe it is better to change the description like this:
The e2e test can be run by cyclone itself. If you deploy with docker, you could run it by typing:
```shell
./scripts/e2e-docker.sh
```
If you deploy with k8s, you could run it by typing:
```shell
./scripts/e2e-k8s.sh
```

Reply:
OK, your description is much better than mine, I will fix it with your suggestion.